exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -189,6 +189,10 @@ func populateOtherDimensions(attributes pcommon.Map, span *Span) {
 			}
 			span.ExternalHttpUrl = value
 			span.HttpUrl = v.Str()
+		} else if k == "http.target" || k == "url.path" {
```

Comment:
let's not update this code as we are not creating these columns in the old table.